models/DataObject/QuantityValue/Unit.php

```diff
@@ -182,26 +184,44 @@ public function __toString()
         return ucfirst($this->getAbbreviation() . ' (' . $this->getId() . ')');
     }
 
+    /**
+     * @param string $abbreviation
+     *
+     * @return Unit
```

Comment:
Should we use this instead for chainable methods?
```suggestion
     * @return $this
```